storybook/manager-head.html

```diff
@@ -15,8 +15,22 @@
   #storybook-explorer-searchfield {
     background-color: white;
   }
+</style>
+
+<script>
+  const storybookDocPath = "kaio-staging"
+  const production = new URLPattern("http{s}?://cultureamp.design");
```

Comment:
Might need to do a regex here since the [URLPattern API](https://developer.mozilla.org/en-US/docs/Web/API/URLPattern) doesn't seem to have great support in firefox and safari

Reply:
Nice one thanks!

Reply:
No worries, I mainly found out cause I had no idea what it was 😆 Looks very handy though! Will be cool when it gets more support

Reply:
+1 to Winter upskilling without asking 😄